config/migrations/2025/20250121103936-extended-sync-op/20250121104105-flush-sync-jobs.sparql

```diff
@@ -0,0 +1,49 @@
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
```

Comment:
This file can be deleted, it's the first time we have an op sync so no previous sync jobs to delete